src/Compilers/Core/Portable/PublicAPI.Unshipped.txt

```diff
@@ -101,6 +101,7 @@ Microsoft.CodeAnalysis.OperationKind.InterpolatedStringText = 83 -> Microsoft.Co
 Microsoft.CodeAnalysis.OperationKind.Interpolation = 84 -> Microsoft.CodeAnalysis.OperationKind
 Microsoft.CodeAnalysis.OperationKind.Invalid = 1 -> Microsoft.CodeAnalysis.OperationKind
 Microsoft.CodeAnalysis.OperationKind.Invocation = 22 -> Microsoft.CodeAnalysis.OperationKind
+Microsoft.CodeAnalysis.OperationKind.IsNull = 89 -> Microsoft.CodeAnalysis.OperationKind
```

Comment:
❗️ This will need to be updated to account for #24401 before the API changes can be properly reviewed.

Reply:
@jasonmalinowski Can we flow the changes from #24401 to all the branches?

Reply:
Oops, there was a branch misconfiguration that meant that didn't flow. Thanks for the catch -- we've reconfigured and it'll flow again tomorrow.